### ISSUE TITLE
TypeError: Cannot read property '$off' of undefined  at g.Bootstrap.destroy

### ISSUE THREAD

**Issue by epacuit** [froala/react-froala-wysiwyg#179]
## Expected Behavior

The FroalaEditor should gracefully mount and unmount when refreshing the page. 

## Actual Behavior
Refreshing a page containing the FroalaEditor generates the following error: 

```
TypeError: Cannot read property '$off' of undefined
    at g.Bootstrap.destroy (froala_editor.min.js:7)
    at t.value (index.js:1)
    at t.value (index.js:1)
    at io (react-dom.production.min.js:202)
    at uo (react-dom.production.min.js:207)
    at mo (react-dom.production.min.js:214)
    at ko (react-dom.production.min.js:220)
    at react-dom.production.min.js:250
    at Object.t.unstable_runWithPriority (scheduler.production.min.js:18)
    at Ko (react-dom.production.min.js:250)
to @ react-dom.production.min.js:198
froala_editor.min.js:7 Uncaught (in promise) TypeError: Cannot read property '$off' of undefined
    at g.Bootstrap.destroy (froala_editor.min.js:7)
    at t.value (index.js:1)
    at t.value (index.js:1)
    at io (react-dom.production.min.js:202)
    at uo (react-dom.production.min.js:207)
    at mo (react-dom.production.min.js:214)
    at ko (react-dom.production.min.js:220)
    at react-dom.production.min.js:250
    at Object.t.unstable_runWithPriority (scheduler.production.min.js:18)
    at Ko (react-dom.production.min.js:250)
index.js:1 Uncaught TypeError: Cannot read property 'on' of undefined
    at t.value (index.js:1)
    at g.Bootstrap.editorInitialized.config.events.initialized (index.js:1)
    at Object.p [as trigger] (froala_editor.min.js:7)
    at g.Bootstrap.n (froala_editor.min.js:7)
```

**Comment by dbx834** [comment]
Anything update on this?

**Comment by abetoots** [comment]
Same behavior when subscribed to Redux store. Connected components that gets updated when subscribed to changes triggers a similar error

**Comment by casualuser** [comment]
fixed in 3.0.4
